language/examples/document-summarization/summarization_with_docai.ipynb

```diff
@@ -0,0 +1,731 @@
+{
```

Comment:
<p>Table formatting in markdown seems broken</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/131/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Hmm, this looks fine to me on ReviewNB and my notebook environment. Might be a GitHub specific thing. I'll try changing the formatting spacing slightly.</p>

Reply:
My change fixed the formatting in the GitHub view